app/views/words/_form.html.erb

```diff
@@ -0,0 +1,16 @@
+    <div class="card" style="width: 10rem; height: 10rem">
+        <div class="card-header">
+            <%= link_to "Dashboard", words_path %>
```

Comment:
change the code to the following:
`<%= link_to "Dashboard", categories_path %>`